src/main/java/bio/terra/service/auth/iam/IamResourceType.java

```diff
@@ -6,10 +6,13 @@
 import org.apache.commons.lang3.StringUtils;
 
 public enum IamResourceType {
+  // To add a hyphenated resource type, special case it in fromEnum and toIamResourceTypeEnum
```

Comment:
Now that we have two cases, maybe it would be better to make the `IamResourceTypeEnum` a field in the enum and initialize it in the constructor:
```
  DATAREPO("datarepo", IamResourceTypeEnum.DATAREPO),
```